apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/ConfigServiceAutoConfiguration.java

```diff
@@ -69,13 +70,16 @@ public GrayReleaseRulesHolder grayReleaseRulesHolder() {
 
   @Bean
   public ConfigService configService() {
+    if (bizConfig.isConfigServiceIncrementalChangeEnabled()) {
```

Comment:
I think the `config-service.cache.enabled` and `config-service.incremental.change.enabled` can be combined.
The first and third scenarios are new.

| config-service.cache.enabled | config-service.incremental.change.enabled |
|------------------------------|-------------------------------------------|
| true                         | true                                      |
| true                         | false                                     |
| false                        | true                                      |
| false                        | false                                     |